README.md

```diff
@@ -114,7 +114,7 @@ The sandboxed containers operator uses three containers:
 
 Container image name | Description | Container repository
 ---------------| ----------- | ----------
- _sandboxed-containers-operator_ |  It contains the controller part of the operator that watches and manages the kataconfig custom resource. It runs as a cluster scoped container. The operator itself is build with operator-sdk. | https://quay.io/isolatedcontainers/sandboxed-containers-operator
+ _openshift-sandboxed-containers-operator_ |  It contains the controller part of the operator that watches and manages the kataconfig custom resource. It runs as a cluster scoped container. The operator itself is build with operator-sdk. | https://quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator
```

Comment:
Still annoyed like for #92 that the quay.io name does not match the spelling for the namespace. Maybe see if you could _also_ get `openshift-sandboxed-containers`? If not, I predict that 100% of our users will get it wrong at least once.

Reply:
I made a request with customer service but it might take a while :-)

Reply:
it's a paid account and getting it was quite a process so I think requesting another one is worse then asking for a rename